Tasks/google-play-promote/package.json

```diff
@@ -9,8 +9,9 @@
   "author": "Jonathan Carter",
   "license": "ISC",
   "dependencies": {
-    "bluebird": "^2.10.1",
-    "googleapis": "^2.1.5",
-    "vsts-task-lib": "2.1.0"
+    "azure-pipelines-task-lib": "2.8.0",
+    "gaxios": "1.8.0",
```

Comment:
This `gaxios` include fixes the build issues I was having when this task ran on Node 6.10.x